packages/components/workbench/src/WorkbenchHeader/WorkbenchHeader.tsx

```diff
@@ -73,15 +90,7 @@ export const WorkbenchHeader = ({
         title
       )}
 
-      {description && (
-        <Paragraph
-          className={styles.description}
-          marginBottom="none"
-          marginRight="spacingM"
-        >
-          {description}
-        </Paragraph>
-      )}
+      {renderDescription && renderDescription(description)}
```

Comment:
I guess it is supposed to be this, right?
```suggestion
      {description && renderDescription(description)}
```